containerized_ml_models/model_api_11/dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM continuumio/anaconda3
+
+COPY api.py .
+COPY model.pkl .
+COPY environment.yml environment.yml
+
```

Comment:
You can use "COPY <src_file_1> <src_file_2> <src_file_3> <dest_folder>" syntax to copy multiple files in a single docker layer. Here it would be "COPY api.py  model.pkl environment.yml /"